tests/lib/workspace.py

```diff
@@ -44,9 +44,13 @@ def __init__(self):
 
         self._generate_keys()
 
-        self._ssh_agent_auth_sock: str = os.path.join(
-            self.working_dir, 'ssh-agent.sock')
-        self._ssh_agent()
+        if config.SSH_USE_EXISTING_AGENT:
+            self._ssh_agent_auth_sock: str = os.getenv('SSH_AUTH_SOCK')
+            logger.info('SSH: Using existing ssh-agent')
```

Comment:
Lets log the socket for clarity

Reply:
Added log message.

When creating one the SSH_AUTH_SOCK is displayed by ssh-agent output